dali/server/daserver.cpp

```diff
@@ -765,3 +767,24 @@ int main(int argc, const char* argv[])
     UseSysLogForOperatorMessages(false);
     return 0;
 }
+
+//
+// Initialize metrics
+void initializeMetrics(IPropertyTree *pConfig)
+{
+    //
+    // Initialize metrics
+    Owned<IPropertyTree> pMetricsTree = pConfig->getPropTree("metrics");
+    if (pMetricsTree == nullptr)
+    {
+        pMetricsTree.setown(getComponentConfigSP()->getPropTree("metrics"));
+    }
+
+    if (pMetricsTree != nullptr)
+    {
+        PROGLOG("Metrics initializing...");
+        hpccMetrics::MetricsManager &metricsManager = hpccMetrics::queryMetricsManager();
+        metricsManager.init(pMetricsTree);
+        metricsManager.startCollecting();
```

Comment:
I don't see a matching call to stopCollecting()

Reply:
Thinking about generalizing this to more components.  Is it worth adding an 

initAndStartCollecting(IPropertyTree * optTree);

function so that instead of this if() block being duplicated in each component it could say

hpccMetrics::queryMetricsManager().initAndStartCollecting(pMetricsTree);

Also, since the code code for getting the pMetricsTree is likely to be identical for each component, even better might be:

hpccMetrics::queryMetricsManager().initAndStartCollecting();